qrscanner/build.gradle

```diff
@@ -3,6 +3,10 @@ apply from: '../buildsystem/constants.gradle'
 apply from: '../buildsystem/helpers.gradle'
 apply from: '../buildsystem/test_dependencies.gradle'
 
+android {
```

Comment:
this can be moved to android_library.gradle, then all modules will have it. I would also define build tool version constant in versions.gradle.